src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -246,6 +246,253 @@ public static void Main()
             }
         }
 
+        [Fact]
+        public void EmitMetadataOnly_XmlDocs_NoDocMode_Success()
+        {
+            CSharpCompilation comp = CreateCompilation(@"
+namespace Goo.Bar
+{
+    /// <summary>This should be emitted</summary>
+    public class Test1
+    {
+        public static void SayHello()
+        {
+            ConEmitMetadataOnly_XmlDocs_NoDocMode_Successsole.WriteLine(""hello"");
```

Comment:
```suggestion
            EmitMetadataOnly_XmlDocs_NoDocMode_Success.WriteLine(""hello"");
```

Reply:
This doesn't look right, aren't we trying to call Console.WriteLine here?

Reply:
sigh...